From ee167e43252bdb9e84cbefcd8b6fe767c9dc768f Mon Sep 17 00:00:00 2001
From: Zhipeng Xie <xiezhipeng1@huawei.com>
Date: Fri, 2 Nov 2018 17:24:15 +0000
Subject: [PATCH 1001/1015] livepatch-patch-hook: don't active patch when
 insmod

we want to active patch after loading the patch.

Signed-off-by: Zhipeng Xie <xiezhipeng1@huawei.com>
---
 kmod/patch/livepatch-patch-hook.c |    7 -------
 1 files changed, 0 insertions(+), 7 deletions(-)

diff --git a/kmod/patch/livepatch-patch-hook.c b/kmod/patch/livepatch-patch-hook.c
index 7a587a3..e59a377 100644
--- a/kmod/patch/livepatch-patch-hook.c
+++ b/kmod/patch/livepatch-patch-hook.c
@@ -443,13 +443,6 @@ static int __init patch_init(void)
 		return ret;
 	}
 
-	ret = klp_enable_patch(lpatch);
-	if (ret) {
-		WARN_ON(klp_unregister_patch(lpatch));
-		patch_free_livepatch(lpatch);
-		return ret;
-	}
-
 	return 0;
 out:
 	patch_free_livepatch(lpatch);
-- 
1.7.5.4

